apps/jupyter/extensions/notebook-runner/src/iframe-api.ts

```diff
@@ -207,6 +215,56 @@ export class EmbeddedPythonCallbacks {
     return undefined;
   }
 
+  async importTask(request: ImportTask["request"]): Promise<undefined> {
+    try {
+      this.setJupyterLocale(request.params.language);
+
+      console.log("starting import task");
+
+      const validateEror = await validateTaskBlob(request.params.task);
+
+      if (validateEror) {
+        console.debug(`${logModule} Importing task`);
+      }
```

Comment:
If I understood correctly, the import should stop if `validateError` returns a non-null value?

Also it may make sense to call this in `detectTaskFormat` and then return `Unknown`?